### PR TITLE
Mar19 glossary tweaks

### PR DESCRIPTION
**Pull request by nfagerlund**

### CODE REVIEW

File: content/source/docs/glossary.html.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ This page collects brief definitions of some of the technical terms used in the
 - [Argument](#argument)
 - [Attribute](#attribute)
 - [Backend](#backend)
-- [Blob Storage](#blob-storage)
 - [Block](#block)
+- [Blob Storage](#blob-storage)
```
Comment (armchairlinguist) [inline content/source/docs/glossary.html.md]: Hm, the original placement looks better to me?

File: content/source/docs/glossary.html.md [hunk 2]
```diff
@@ -871,9 +895,11 @@ Available Terraform versions are configured at a per-instance level in [Private
 
 Also "input variables".
 
-Key/value pairs used in a [run][]. Terraform [modules][] can declare variables to allow customization. For child modules, the parent module provides a value when calling the module; for the [root module][], values are provided at run time.
+In general-purpose programming, a variable is a symbolic name associated with a value.
+
+In Terraform, "variables" almost always refers to _input variables,_ which are key/value pairs used in a [run][]. Terraform [modules][] can declare variables to allow customization. For child modules, the parent module provides a value when calling the module; for the [root module][], values are provided at run time.
 
-TFE lets you set root input variables in a [workspace][], so all collaborators can use the same values. Variable values marked as "sensitive" become unreadable in the UI and API, and all variable values are protected by Vault.
+TFE lets you set values for root input variables in a [workspace][], so all collaborators can use the same values. Variable values marked as "sensitive" become unreadable in the UI and API, and all variable values are protected by Vault.
```
Comment (armchairlinguist) [inline content/source/docs/glossary.html.md]: I wonder if it's worth linking out to the other docs on this?

File: content/source/docs/glossary.html.md (hunk 1 above)
Comment (nfagerlund) [reply content/source/docs/glossary.html.md]: UGH you're right 😩 humans shouldn't be in charge of alphabetizing anyway 😤

File: content/source/docs/glossary.html.md (hunk 2 above)
Comment (nfagerlund) [reply content/source/docs/glossary.html.md]: There's a link to the TFE workspace variables docs right below!

**Comment by nfagerlund** [comment]
@armchairlinguist Cool to merge on this?

**Pull request merged**